src/foam/nanos/auth/User.js

```diff
@@ -322,7 +322,7 @@ foam.CLASS({
       gridColumns: 6
     },
     {
-      class: 'Boolean',
+      class: 'Boolean',value:true,
```

Comment:
Why cram onto one line?

Reply:
oops that's not supposed to be there, just added to bypass verifying email when i test
```suggestion
```